app/views/users/show.html.erb

```diff
@@ -1 +1,11 @@
-<h1>Here is information about an user</h1>
+<%= render 'partials/user_card', user: @user %>
+<div class='user-bio'>
+  <h3>Bio</h3>
+  <h3><%= @user.bio %></h3>
+</div>
+<ul class='thre-posts-list'>
+  <% @posts.each_with_index do |post, index| %>
+    <li><%= render 'partials/post_card', post: post, index: index %></li>
+  <% end %>
+</ul>
+<button class='all-posts-button'>See all posts</button>
```

Comment:
- [ ] Kindly make sure the **See all Post** button/link opens user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png). i.e when clicking the the **See all Post** button/link, it should display this **[wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)**